### DISCUSSION TITLE
Should translations be turned off for error messages that are simple enough?

### DISCUSSION THREAD

**Discussion by NickSchmitt** [mattpocock/ts-error-translator#35] [Ideas]
Might it be worth turning translation off for error messages that are deemed "simple enough"? (whatever might determine that)

E.g. instead of

```
Expected 3 arguments, but got 2.
No Translation Found
Could not find a translation for error code #2554
```
just
```
Expected 3 arguments, but got 2.
```

**Comment by mattpocock** [comment]
I've seen folks confused by what this argument means, so I don't think so!
